### ISSUE TITLE
Feature request: Friendlier date picker

### ISSUE THREAD

**Issue by jenniferlynparsons** [stom79/FediPlan#2]
If it's possible, a GUI, more user-friendly date picker would be a welcome addition!

**Comment by stom79** [comment]
Yes, UI is currently really basic. But the date picker should be available. On what browser did you try Fediplan?

**Comment by jenniferlynparsons** [comment]
Sorry for the long delay! I was using Firefox on a Mac.

**Comment by bugsysop** [comment]
No date picker also with Firefox on GNU/Linux
